src/jsglue.cpp

```diff
@@ -851,26 +849,19 @@ static size_t MallocSizeOf(const void* aPtr)
 #endif
 }
 
-
 bool
 CollectServoSizes(JSRuntime *rt, JS::ServoSizes *sizes, GetSize gs)
 {
-    mozilla::PodZero(sizes);
-    bool (*pass)(JSObject* obj, nsISupports** iface);
-    pass = &pass_to_ctor;
-    
-    if(pass == nullptr){
-      return false;
-    }
+  mozilla::PodZero(sizes);
 
-    ObjectPrivateVisitorHeap * opvh = new ObjectPrivateVisitorHeap(gs, pass);
+  ServoDOMVisitor * opvh = new ServoDOMVisitor(gs, ShouldMeasureObject);
```

Comment:
Oh, one other thing - this is a memory leak. What if you make it a stack allocation and pass `&opvh` instead.

Reply:
Ok, will do. With regard to squashing, I am working on it, I have to resolve some conflict of my own creation. Sorry about that.